.github/workflows/coverity.yml

```diff
@@ -0,0 +1,41 @@
+
+name: Coverity
+on:
+  schedule:
+    - cron: '0 5 * * *' # Daily at 05:00 UTC
+
+jobs:
+  coverity:
+    name: "Test Suite"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: cockpit-composer
+    steps:
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+         path: cockpit-composer
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install nodejs npm
+
+      - name: Download Coverity Tool
+        run: |
+          make coverity-download
+        env:
+         COVERITY_TOKEN: ${{ secrets.COVERITY_TOKEN }}
+         COVERITY_EMAIL: ${{ secrets.COVERITY_EMAIL }}
```

Comment:
The `COVERITY_EMAIL` is not really needed here. I dropped it for osbuild.

Reply:
Okay. Removed.